service.py

```diff
@@ -70,12 +70,13 @@ def org_user(org):
                         logger.info(f"Organization invitation sent to username: {invi_username}")
                     if decoded_data.get('state') == "active":
                         logger.info(f"User with username: {invi_username}, already part of organization")
-                    
-                if invi_response.status_code == 422 or invi_response.status_code == 403:
+                elif invi_response.status_code == 422 or invi_response.status_code == 403:
                     logger.warning(f"Organization invitation could not be sent to username: {invi_username}")
                     logger.warning(f"Failed with error code: {invi_response.status_code}")
-
+                else:
```

Comment:
as an example - status 201 would fall here as well as 404.
logging based on `invi_response.ok` would eliminate conflicting log messages at least.

Reply:
Changed it now to also handle ok, but unexpected return codes